client/src/components/application/Examine/nwpta/nwpta.vue

```diff
@@ -189,6 +229,9 @@
       else this.requested_radio_selection = 'requested_nuans';
     },
     methods: {
+      logit() {
+        console.log('it happened')
```

Comment:
Please remove any console.log() statements that are clearly for debugging and not descriptive enough to help production debugging ('got here', '***', that kind of thing). Feel free to leave the ones that output, say, "got into ABC function" or an object that might be useful down the road for production debugging.

Reply:
oops, yes, I meant to delete that one.